app/models/game.rb

```diff
@@ -0,0 +1,9 @@
+class Game < ActiveRecord::Base
+  has_many :memberships
+  has_many :users, through: :memberships
+
+  def <<(user)
```

Comment:
Makes sense to me. I'll change it to `add_user` and if implementing enumerability is appropriate in the future I'll revisit the idea.